emottak-utils/src/main/kotlin/no/nav/emottak/utils/events/model/Event.kt

```diff
@@ -0,0 +1,20 @@
+package no.nav.emottak.utils.events.model
+
+import kotlinx.serialization.Serializable
+import no.nav.emottak.utils.UuidSerializer
+import kotlin.uuid.ExperimentalUuidApi
+import kotlin.uuid.Uuid
+
+@OptIn(ExperimentalUuidApi::class)
+@Serializable
+data class Event(
+    // @Serializable(with = UuidSerializer::class)
+    // val eventId: Uuid,                   // Genereres ved insert til DB
+    val eventType: EventType,
+    @Serializable(with = UuidSerializer::class)
+    val requestId: Uuid,
+    val contentId: String? = null,
+    val messageId: String,
+    val eventData: String? = null // TODO: ByteArray? Eller JSON? Eller String?
+    // val createdAt: Instant? = null       // Genereres ved insert til DB
```

Comment:
```suggestion
    val eventType: EventType,
    @Serializable(with = UuidSerializer::class)
    val requestId: Uuid,
    val contentId: String? = null,
    val messageId: String,
    val eventData: String? = null
```

Denne bør sikkert tas opp i på Slack i stedet for å slenge spørsmål i koden: `// TODO: ByteArray? Eller JSON? Eller String?` :smile:

Reply:
Enig i at det ikke trenger å legges inn som en TODO, men det er fulgt opp her: https://github.com/navikt/ebxml-processor/pull/111#issuecomment-2694820264